test/elixir_boilerplate_web/errors_test.exs

```diff
@@ -63,15 +63,11 @@ defmodule ElixirBoilerplateWeb.ErrorsTest do
       |> User.changeset(%{"email" => "foo", "nicknames" => [], "single_role" => %{"type" => "bar"}, "multiple_roles" => [%{"type" => ""}]})
       |> changeset_to_error_messages()
 
-    assert html == """
-             <ul>
-                 <li>email has invalid format</li>
-                 <li>email should be 10 characters</li>
-                 <li>multiple_roles.type can’t be blank</li>
-                 <li>nicknames should have at least 1 item</li>
-                 <li>single_role.type is invalid</li>
-             </ul>
-           """
+    assert html =~ "<li>email has invalid format</li>"
+    assert html =~ "<li>email should be 10 characters</li>"
+    assert html =~ "<li>multiple_roles.type can’t be blank</li>"
+    assert html =~ "<li>nicknames should have at least 1 item</li>"
+    assert html =~ "<li>single_role.type is invalid</li>"
```

Comment:
What is the link between Elixir 1.11 and this? The pull request description doesn’t mention it.

Reply:
It looks like a collateral from `1.11`…

There are extra `\n` in the generated _safe html_ and modifying the heredoc string to add extra blank lines felt like a patch more than an test intend!

Reply:
This could be a good candidate to introduce our own HTML testing libs (https://github.com/mirego/html_test_helpers and https://github.com/mirego/html_test_identifiers). These were created for these purpose exactly and would add traction and usage of the libs by the community!

Reply:
To me, adding libs that are not published to [hex.pm](https://hex.pm/) is like including _beta_ dependencies, I don't see how this would be relevant… 

Especially for the intend of a single test that validates the html serialization of an error changeset!

Reply:
I’ve added the `hex` badge in both repos to prevent confusion.

* https://github.com/mirego/html_test_identifiers/pull/5
* https://github.com/mirego/html_test_helpers/pull/5